tests/test-basic-user.sh

```diff
@@ -19,10 +19,16 @@
 
 set -euo pipefail
 
-echo "1..1"
-
 . $(dirname $0)/libtest.sh
 
+touch test-xattrs
+if ! setfattr -n user.testvalue -v somevalue test-xattrs; then
```

Comment:
Maybe extract this into a `libtest.sh` function like `skip_without_user_xattrs` ?